1_fetch/src/my_happy_script_ms_11-12-21.R

```diff
@@ -0,0 +1,196 @@
+##########################
+# Model_RMSE data Plot   #
+#                        #
+# Author:                #     
+# Date:                  #
+##########################
+
+
+## Libs
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+
+## Check that you are in project home directory - should be in project home directory
+print(getwd())
+
+## Global Variables:
+
+in_path = '1_fetch/src' 
+out_path = '1_fetch/out'
```

Comment:
The script will not currently run if someone clones your repository and tries to run the code, because this directory isn't in git. One approach would be to write code to check whether the directory exists, and create it if it doesn't. But it is more elegant just to ensure that the directory shows up for others by committing a `.empty` file as detailed in my general comment.

Reply:
hmm, i realized I wrote in my `.gitignore` `*/out/*` to exclude all files in out folder. Wouldn't that mean that even if I include a `.empty` in the `out` folder, the `out` will not appear in the repo? 
Instead, I can just add replace `*/out/*` in my gitignore replace it with calls to exclude the specific output files.

Reply:
Yes, that works. But it does require you to potentially keep editing the `.gitignore` as you create outputs of new names. Another option is to keep the `*/out/*` in your `.gitignore` and then force git to commit the `.empty` file anyways by adding the `-f` flag like this: `git commit -f 1_fetch/out/.empty`